section4/exercises/classMethods.js

```diff
@@ -28,13 +28,40 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
```

Comment:
@bhohnco - I believe you would benefit from taking another look at this `Burrito` exercise, particularly regarding the instructions for the `changeProtein` and `addTopping` methods. You created and named the methods correctly, yes, but as the directions point out, the `changeMethod` should also accept one argument (a string) and the method should re-assign the `this.protein` attribute to the value passed in as an argument:

```
// The burrito class should have a method named `changeProtein`. 
// This method should accept one argument, a String.
// The method should re-assign this.protein to the value that was passed in.
```

Can you review and update both your methods below?

Reply:
@mschae16 
Thanks for the feedback. I see what I did wrong here and figured out how to adjust it. The changes I made to the methods is below, and I also attached a screenshot of the adjusted code. Please let me know if you need me to re-push/commit these changes. 

  }
  changeProtein (switchProtein) {
    this.protein = switchProtein
  };
  addTopping(topping) {
  this.toppings = (this.toppings + topping);
  };
![Screen Shot 2020-11-11 at 3 09 08 PM](https://user-images.githubusercontent.com/71860165/98870468-91ea1780-2430-11eb-9056-cc0d2ec7e7cf.png)

Reply:
@bhohnco - Thanks for taking another look. Your updates to the `changeProtein` method looks good in terms of re-assigning the attribute with the new argument.  However, could you take another look at the `addTopping` method?  I believe the attribute `this.toppings` is of data type **array** and it looks like you're using concatenation (which is for strings). Can you think of any functions or methods that you've learned that can add an item into an array of items?

Reply:
Also please feel free to go ahead and commit and push up your updates, thanks!

Reply:
@mschae16 
Taking another look at this, I should have used the push method for the toppings to add an additional topping to the array of normal toppings. After adjusting this, it looks much cleaner in my code. I've also added this to the tweet assignment for adding comments to a post. I've committed and push the update to my mod1 pre-work.  Thanks for the feedback!